### PR TITLE
fix typos and docs standards

### PR DESCRIPTION
**Pull request by akkspros**
fix typos and docs standards

### CODE REVIEW

**Comment by akkspros** [comment]
@schlessera can you check this please

File: php/WP_CLI/Compat/FeedbackMethodTrait.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace WP_CLI\Compat;
 
-// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound,Generic.Classes.DuplicateClassName.Found
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound,Generic.Classes.DuplicateClassName.Found.
```
Comment (schlessera) [inline php/WP_CLI/Compat/FeedbackMethodTrait.php]: PHPCodeSniffer directives shouldn't end in a comma. Please revert this change.

File: php/WP_CLI/Compat/Min_PHP_5_6/FeedbackMethodTrait.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ trait FeedbackMethodTrait {
 	 * @param mixed  ...$args Optional text replacements.
 	 *
 	 */
-	public function feedback( $string, ...$args ) { // phpcs:ignore PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ellipsisFound
+	public function feedback( $string, ...$args ) { // phpcs:ignore PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ellipsisFound.
```
Comment (schlessera) [inline php/WP_CLI/Compat/Min_PHP_5_6/FeedbackMethodTrait.php]: PHPCodeSniffer directives shouldn't end in a comma. Please revert this change.

File: php/WP_CLI/ComposerIO.php [hunk 3]
```diff
@@ -33,7 +33,7 @@ public function writeError( $messages, $newline = true, $verbosity = self::NORMA
 
 	private static function output_clean_message( $message ) {
 		$message = preg_replace( '#<(https?)([^>]+)>#', '$1$2', $message );
-		// phpcs:ignore WordPress.WP.AlternativeFunctions.strip_tags_strip_tags
+		// phpcs:ignore WordPress.WP.AlternativeFunctions.strip_tags_strip_tags.
```
Comment (schlessera) [inline php/WP_CLI/ComposerIO.php]: PHPCodeSniffer directives shouldn't end in a comma. Please revert this change.
